src/t2iapi/metric/service.proto

```diff
@@ -115,4 +115,12 @@ service MetricService {
    */
   rpc TransitionProvideValueToExpressNoValueAvailable (BasicHandleRequest)
       returns (BasicResponse);
+
+  /*
+  Convey demo values for the metric with the given handle.
+  This means that the metric source is part of demo data and the metric value is arbitrary and for demonstration
```

Comment:
I don't understand `the metric source is part of demo data`, what is that supposed to mean? Is it even necessary to express that? Why not

```suggestion
  This means that the metric value is arbitrary and for demonstration
```

Reply:
The wording was a mix of the given Biceps description. I changed this, shorter is better.
done